source/server/backtrace.h

```diff
@@ -83,6 +83,23 @@ class BackwardsTrace : Logger::Loggable<Logger::Id::backtrace> {
   }
 
   /**
+   * Capture a stack trace from a particular address.
+   *
+   * This can be used to capture a useful stack trace from a fatal signal
+   * handler.
+   *
+   * @param address The stack trace will begin from this address.
+   */
+  void CaptureFrom(void* address) {
+#ifdef NDEBUG
```

Comment:
In practice, do we use any of the log in prod stuff anymore? Can we just kill it?